src/templates/finops-hub/modules/dataFactory.bicep

```diff
@@ -981,476 +981,480 @@ resource trigger_MonthlySchedule 'Microsoft.DataFactory/factories/triggers@2018-
 // config_InitializeHub pipeline
 //------------------------------------------------------------------------------
 @description('Initializes the hub instance based on the configuration settings.')
-resource pipeline_InitializeHub 'Microsoft.DataFactory/factories/pipelines@2018-06-01' = if (deployDataExplorer) {
-  name: '${safeConfigContainerName}_InitializeHub'
-  parent: dataFactory
-  properties: {
-    activities: [
-      { // Get Config
-        name: 'Get Config'
-        type: 'Lookup'
-        dependsOn: []
-        policy: {
-          timeout: '0.00:05:00'
-          retry: 2
-          retryIntervalInSeconds: 30
-          secureOutput: false
-          secureInput: false
-        }
-        userProperties: []
-        typeProperties: {
-          source: {
-            type: 'JsonSource'
-            storeSettings: {
-              type: 'AzureBlobFSReadSettings'
-              recursive: true
-              enablePartitionDiscovery: false
+module pipeline_InitializeHub 'hub-pipeline.bicep' = if (deployDataExplorer) {
+  name: 'pipeline_InitializeHub'
+  params: {
+    dataFactoryName: dataFactory.name
+    pipelineName: '${safeConfigContainerName}_InitializeHub'
+    pipelineProperties: {
+      activities: [
```

Comment:
This is mostly spacing changes. To hide spacing changes, use the gear menu above the files 👆 and select "Hide whitespace", then apply and reload.